pkg/logs/config/config_test.go

```diff
@@ -405,7 +405,7 @@ func (suite *ConfigTestSuite) TestMultipleTCPEndpointsInConf() {
 		CompressionLevel: 0,
 		ProxyAddress:     "proxy.test:3128"}
 
-	expectedEndpoints := NewEndpoints(expectedMainEndpoint, []Endpoint{expectedAdditionalEndpoint}, true, false)
+	expectedEndpoints := NewEndpoints(expectedMainEndpoint, nil, []Endpoint{expectedAdditionalEndpoint}, true, false)
```

Comment:
Do we have any other tests here w/ backup endpoint defined?

Reply:
Good call - ill add one